Vagrantfile

```diff
@@ -7,29 +7,13 @@ Vagrant.configure(2) do |config|
   config.vm.network "forwarded_port", guest: 5000, host: 5001
   config.vm.synced_folder ".", "/vagrant", type: "sshfs"
```

Comment:
By the way, @dustymabe just got vagrant-sshfs into Bodhi today, so soon you won't need to set up the copr anymore. Pretty sweet, right?